docs/build-the-docs.md

```diff
@@ -388,3 +386,12 @@ to delete the `cache` directory with the command below (use the error path print
 ```
 rm -r /var/owncloud/docs/cache
 ```
+
+### Manually Restarting CI
+
+In case CI needs to be restarted, which can happen in the rare case it was not triggered post pushing automatically, you need to manually (re)start the CI. This can be done by creating an empty commit and pushing it. To do so, change to the branch in question and follow the git commands below:
+
+```
+git commit --allow-empty -m "restart ci"
+gip push -f
```

Comment:
```suggestion
git push
```

If just adding a commit, like this, then `git push` should be enough. The local branch will be cleanly 1 commit ahead of the upstream at GitHub.